dmriqcpy/viz/utils.py

```diff
@@ -32,11 +32,11 @@ def analyse_qa(stats_per_subjects, stats_across_subjects, column_names):
         for name in stats_per_subjects.index:
             if stats_per_subjects.at[name, metric] > mean + 2 * std or\
                stats_per_subjects.at[name, metric] < mean - 2 * std:
-                warning[metric].append(name[0])
```

Comment:
Why did you change this ?

Reply:
I think, it is easier to read.

Reply:
I used this so instead of reading the whole *.bval it use the path+baseline but without extension.